www/lendingbot.js

```diff
@@ -370,6 +376,30 @@ function doSave() {
     });
     localStorage.setItem('timespanNames', JSON.stringify(timespanNames))
 
+    // Bitcoin Display Unit
+    localStorage.displayUnitText = $('input[name="btcDisplayUnit"]:checked').val();
```

Comment:
Also just noticed, I believe this can be refactored:

```
    validOutputCurrencyDisplayModes.forEach(function(mode) {
        if(mode == localStorage.outputCurrencyDisplayModeText) {
            outputCurrencyDisplayMode = mode;
        }
    })
```

to

```
    if(validOutputCurrencyDisplayModes.indexOf(mode) !== -1) {
        outputCurrencyDisplayMode = mode;
    }
```